chainstate/src/detail/tests/normal_tokens.rs

```diff
@@ -0,0 +1,837 @@
+        // let mut chainstate = setup_chainstate();
+        let values = vec![OutputValue::Token(TokenData::TokenIssuanceV1 {
+            token_ticker: b"USDC".to_vec(),
+            amount_to_issue: Amount::from_atoms(52292852472),
+            number_of_decimals: 1,
+            metadata_uri: b"https://some_site.meta".to_vec(),
+        })];
+        let block_index =
+            process_token(&mut test_framework, ParentBlock::BestBlock, values.clone())
+                .unwrap()
+                .unwrap();
+        let block = test_framework.block(*block_index.block_id());
+        assert_eq!(block.transactions()[0].outputs()[0].value(), &values[0]);
+
+        // Ticker is too long
+        let values = vec![OutputValue::Token(TokenData::TokenIssuanceV1 {
+            token_ticker: b"TRY TO USE THE LONG NAME".to_vec(),
+            amount_to_issue: Amount::from_atoms(52292852472),
+            number_of_decimals: 1,
+            metadata_uri: b"https://some_site.meta".to_vec(),
+        })];
+        assert!(matches!(
+            process_token(&mut test_framework, ParentBlock::BestBlock, values),
+            Err(BlockError::CheckBlockFailed(
+                CheckBlockError::CheckTransactionFailed(CheckBlockTransactionsError::TokensError(
+                    TokensError::IssueErrorIncorrectTicker(_, _)
+                ))
+            ))
+        ));
+
+        // Doesn't exist ticker
+        let values = vec![OutputValue::Token(TokenData::TokenIssuanceV1 {
+            token_ticker: b"".to_vec(),
+            amount_to_issue: Amount::from_atoms(52292852472),
+            number_of_decimals: 1,
+            metadata_uri: b"https://some_site.meta".to_vec(),
+        })];
+        assert!(matches!(
+            process_token(&mut test_framework, ParentBlock::BestBlock, values),
+            Err(BlockError::CheckBlockFailed(
+                CheckBlockError::CheckTransactionFailed(CheckBlockTransactionsError::TokensError(
+                    TokensError::IssueErrorIncorrectTicker(_, _)
+                ))
+            ))
+        ));
+
+        // Ticker contain not alpha-numeric byte
+        let values = vec![OutputValue::Token(TokenData::TokenIssuanceV1 {
+            token_ticker: "💖".as_bytes().to_vec(),
+            amount_to_issue: Amount::from_atoms(52292852472),
+            number_of_decimals: 1,
+            metadata_uri: b"https://some_site.meta".to_vec(),
+        })];
+        assert!(matches!(
+            process_token(&mut test_framework, ParentBlock::BestBlock, values),
+            Err(BlockError::CheckBlockFailed(
+                CheckBlockError::CheckTransactionFailed(CheckBlockTransactionsError::TokensError(
+                    TokensError::IssueErrorIncorrectTicker(_, _)
+                ))
+            ))
+        ));
+
+        // Issue amount is too low
+        let values = vec![OutputValue::Token(TokenData::TokenIssuanceV1 {
+            token_ticker: "USDT".as_bytes().to_vec(),
```

Comment:
Same...

Reply:
Okay there's a bunch of these. It applied to all.

Reply:
Ok, I rid off all USDT and USDC mentions. Thanks!